### ISSUE TITLE
"Parameter is never cleared or assigned" in CALL TRANSFORMATION block

### ISSUE THREAD

**Issue by cyb3rko** [SAP/abap-cleaner#267]
Hello there,  
I have the following method:
``` abap
METHOD serialize.
  " TODO: parameter RESULT is never cleared or assigned (ABAP cleaner)

  CALL TRANSFORMATION id
       SOURCE values = object
       RESULT XML result.
ENDMETHOD.
```

---

ABAP cleaner gives the warning that `result` is never assigned, even though the call of "CALL TRANSFORMATION ... RESULT XML result" assigns a new value to `result`.

**Comment by jmgrassau** [comment]
Hi Niko, 

thanks a lot for reporting this! This bug will be fixed with the next release.

Kind regards,
Jörg-Michael

**Comment by jmgrassau** [comment]
Hi Niko, 

this bug should now be fixed with version 1.15.0, which was just released! Thanks again for opening this issue.

Kind regards,
Jörg-Michael
